.gitlab-ci.yml

```
@@ -471,15 +471,15 @@ test:helm_chart_upgrade_from_current_lts_to_current_release:
   script:
     - tests/ci-make-deps.sh
     - |
-      echo "INFO - installing mender tag ${MENDER_CURRENT_LTS_RELEASE_TAG:-mender-3.6.3}"
+      echo "INFO - installing mender tag ${MENDER_CURRENT_LTS_RELEASE_TAG:-mender-3.6.5}"
```

This is supposed to be the current LTS:
```suggestion
      echo "INFO - installing mender tag ${MENDER_CURRENT_LTS_RELEASE_TAG:-mender-3.7.5}"
```
This is for testing the upgrade from an already release helm chart (chart.mender.io) to the local helm chart.

---

But then both the old and the new will be 3.7.5, so it will be upgrading to the same version it came from. Is that correct?

---

They are the same Mender version (this time, not the next non-LTS release), but most importantly, they are two different Helm Chart versions: here the test is focusing on the Helm code, not on Mender versions:

Line 488: the first installation installs the current LTS with the latest charts.mender.io chart (`--version` is not specified, so it get the latest`): 
```
        mender/mender || exit 3;
```
Line 504: the test upgrades to the latest Helm code, present in `master`, which is still not published in the charts.mender.io; mender will be the latest release (LTS or not)
```
        ./mender || exit 3;
```

---

Aha, that makes sense!